sdk/documentintelligence/azure-ai-documentintelligence/samples/README.md

```diff
@@ -0,0 +1,97 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-cognitive-services
+  - azure-document-intelligence
+urlFragment: documentintelligence-samples
+---
+
+# Samples for Azure Document Intelligence client library for Python
+
+These code samples show common scenario operations with the Azure Document Intelligence client library.
+
+All of these samples need the endpoint to your Document Intelligence resource ([instructions on how to get endpoint][get-endpoint-instructions]), and your Document Intelligence API key ([instructions on how to get key][get-key-instructions]).
+
+|**File Name**|**Description**|
```

Comment:
Let's just simplify this file for today's release and remove this table. Add a link to the samples directory instead. Open an issue for yourself to fix this before the next release.